### DISCUSSION TITLE
Welcome to H96_MAX_V58 Discussions!

### DISCUSSION THREAD

**Discussion by ncravino** [ncravino/H96_MAX_V58#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DRoca-Leonardo** [comment]
# [ANN] Custom LineageOS 23.2 (Android 16) / Kernel 6.1 for H96 Max v58 (Kiosk Mode / Non-TV)

Hi everyone,

I have developed a highly optimized custom system for the **H96 Max v58** TV Box, designed specifically for **self-service kiosks, interactive totems, and digital signage (NOT for conventional Android TV use)**. Google Apps and Service Working 100%

### 🚀 Key Features:
* **OS Base:** LineageOS 23.2 powered by **Android 16**.
* **Upgraded Kernel:** Running on **LTS Kernel 6.1**.
* **Clean Architecture:** Completely removed all `vendor.img` dependencies.
* **Direct Hardware Control:** Low-level hardware components are managed directly via the kernel, achieving 100% stability and peak performance for commercial, high-uptime environments.
* **Status:** 100% functional and production-ready.

---

📥 **Contact & Inquiries:**
If you require this deployment for your projects, need commercial support, or are interested in implementing this kiosk solution, please **contact me directly through this platform** (by opening a discussion here, via private message, or checking my profile details).

---
*Tags: #H96MaxV58 #Android16 #LineageOS23 #Kernel61 #KioskMode #EmbeddedAndroid #Rockchip*
